src/components/member-card/card.module.scss

```diff
@@ -14,6 +14,10 @@
   height: 50px;
 }
 
+.removedPointerEvent {
+  pointer-events: none;
+}
+
```

Comment:
we can better the class name here. It's synonymous with the event its doing now.